pkg/transfer/transfer.go

```diff
@@ -61,7 +62,7 @@ type ImageFilterer interface {
 // the provided descriptor. The descriptor may be any type of manifest
 // including an index with multiple image references.
 type ImageStorer interface {
-	Store(context.Context, ocispec.Descriptor, images.Store) ([]images.Image, error)
+	Store(context.Context, ocispec.Descriptor, images.Store, events.Publisher) ([]images.Image, error)
```

Comment:
should we be modifying the interface here or should the implementor of `ImageStorer` be passed a reference to the publisher when it is created, much like the transfer service gets one

Reply:
The rest of the interfaces for transfer service in pkg/transfer/transfer.go are being passed the content or image store references as need. So to maintain uniformity, passed the events publisher as well.

@dmcgowan any preferences to how this is done? Or does this look ok?